data/afreen0102.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Afreen Syed",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " Web Developer | Open Source Contributer | Ex-SDE Intern @TigerAnalytics | Coder | Communities",
+  "avatar" : "https://pbs.twimg.com/profile_images/1618252955773198336/Na8GobmE_400x400.jpg",
```

Comment:
We recommend the GitHub profile picture (which is your GitHub username `.png`  as this is on the approved domains. 

As this is not a blocker I will merge this PR